docs/modules/ROOT/pages/introduction.adoc

```diff
@@ -66,7 +66,7 @@ Additionally, prerelease version numbers may have additional suffixes, for examp
 [[introduction-requirements]]
 == Requirements
 
-1. https://neo4j.com/[Neo4j Database] 4.3.x (with `x` being latest patch) with https://neo4j.com/developer/neo4j-apoc/[APOC] plugin:
+1. https://neo4j.com/[Neo4j Database] version 4.3.x, 4.4.x (with `x` being latest patch) or 5.x (with `x` being latest minor) with https://neo4j.com/developer/neo4j-apoc/[APOC] plugin:
```

Comment:
Would it be more compact and clever going forward to just write down the minimum Neo4j version requirement rather than pass the whole list?
For instance `Neo4j version >= 4.3.0` or literal `Neo4j GraphQL library supports Neo4j version 4.3.0 and newer`.
I would remove also the `("x" being latest patch/minor)` seems to me a little condescending to the reader, in my opinion!